apps/web/src/data/subgraph/requests/exploreQueries.ts

```diff
@@ -9,28 +10,36 @@ import {
   ExploreDaoFragment,
   OrderDirection,
 } from '../sdk.generated'
+import { MyDaosResponse } from './daoQuery'
+
+export type ExploreDaoWithChainId = ExploreDaoFragment & { chainId?: CHAIN_ID }
 
 export interface ExploreDaosResponse {
-  daos: ExploreDaoFragment[]
+  daos: ExploreDaoWithChainId[]
   hasNextPage: boolean
 }
 
 export const userDaosFilter = async (
-  chainId: CHAIN_ID,
   memberAddress: string
 ): Promise<ExploreDaosResponse | undefined> => {
-  const first = 30
-  const userDaos = await SDK.connect(chainId).daoTokenOwners({
-    where: {
-      owner: memberAddress,
-    },
-    first,
-  })
-
-  const daoAddresses = userDaos.daotokenOwners.map((x) => x.dao.tokenAddress)
-  const data = await SDK.connect(chainId).myDaosPage({ daos: daoAddresses })
-
-  return { daos: data.auctions, hasNextPage: data.auctions.length === first }
+  const userDaos = await axios
```

Comment:
Lots of decently complex logic here. Perhaps some error handling in case something goes wrong? Would be easier to narrow down when debugging.

Reply:
Also, filtering is one among many things happening here. Is userDaosFilter the best name for this function?